Movecraft/src/main/java/net/countercraft/movecraft/async/rotation/RotationTask.java

```diff
@@ -166,13 +166,13 @@ protected void execute() {
 
         // Rotates the craft's tracked locations, and all parent craft's.
         Craft temp = craft;
-        do {
-            for (Set<TrackedLocation> locations : craft.getTrackedLocations().values()) {
+        //do {
+            for (Set<TrackedLocation> locations : temp.getTrackedLocations().values()) {
                 for (TrackedLocation location : locations) {
                     location.rotate(rotation, originPoint);
                 }
             }
-        } while (temp instanceof SubCraft && (temp = ((SubCraft) temp).getParent()) != null);
+        //} while (temp instanceof SubCraft && (temp = ((SubCraft) temp).getParent()) != null);
```

Comment:
Does this no longer need to handle recursively deep subcrafts?

Reply:
Its no longer necessary as the trackedlocations will be transferred to the subcraft and back to the parent whenever a subceaft is created. That results in there never being a trackedlocation on the parent for which the resulting/effective location is actually aboard a subcraft